neo4j-ogm-docs/src/main/asciidoc/reference/nativetypes.adoc

```diff
@@ -0,0 +1,183 @@
+Those are not the Java types with the same name but Neo4j specific types that map to `long` and `double` respectively.
+Please refer to both the https://neo4j.com/docs/cypher-manual/current/syntax/values/[Cypher] and https://neo4j.com/docs/developer-manual/current/drivers/cypher-values/[Java-Driver manual] for further information about the type system.
+
+While you have to take a bit of care when modelling entities with numeric attributes (in regards of precession and scale),
+mapping of numbers, strings and boolean attributes is pretty much straight forward.
+Temporal and spatial types however made their first appearances in Neo4j 3.4.
+Therefore OGM provided <<reference:type-conversion:custom, type conversion>> for those to store them as string or numeric types.
+Especially it maps all temporal types onto ISO 8601 formatted strings and spatial types onto a composite, map based structure.
+
+Starting with Neo4j-OGM 3.2, OGM provides dedicated support for Neo4j's temporal and spatial types.
+
+[[reference:native-property-types:supported-drivers]]
+== Supported drivers
+
+Neo4j-OGM supports all Neo4j temporal and spatial types for the Bolt and the embedded driver.
+Please add the following module if you're using the Bolt driver:
+
+.Support for native types temporal and spatial types with the Bolt driver
+[source, xml, subs="attributes, specialcharacters"]
+----
+<dependency>
+    <groupId>org.neo4j</groupId>
+    <artifactId>neo4j-ogm-bolt-driver</artifactId>
+    <version>{ogm-version}</version>
+    <scope>runtime</scope>
+</dependency>
+
+<dependency>
+    <groupId>org.neo4j</groupId>
+    <artifactId>neo4j-ogm-bolt-native-types</artifactId>
+    <version>{ogm-version}</version>
+    <scope>runtime</scope>
+</dependency>
+----
+
+The setup for the embedded driver looks similar:
+
+.Support for native types temporal and spatial types with the embedded driver
+[source, xml, subs="attributes, specialcharacters"]
+----
+<dependency>
+    <groupId>org.neo4j</groupId>
+    <artifactId>neo4j-ogm-embedded-driver</artifactId>
+    <version>{ogm-version}</version>
+    <scope>runtime</scope>
+</dependency>
+
+<dependency>
+    <groupId>org.neo4j</groupId>
+    <artifactId>neo4j-ogm-embedded-native-types</artifactId>
+    <version>{ogm-version}</version>
+    <scope>runtime</scope>
+</dependency>
+
+<dependency>
+    <groupId>org.neo4j</groupId>
+    <artifactId>neo4j</artifactId>
+    <version>{neo4j-version}</version>
+    <scope>runtime</scope>
+</dependency>
+----
+
+The dependencies for a Gradle build are `'org.neo4j:neo4j-ogm-bolt-driver:{ogm-version}'` and  `'org.neo4j:neo4j-ogm-embedded-native-types:{ogm-version}'` respectively.
+
+[[reference:native-property-types:optin]]
+== Opt-in to use native types
+
+Using native types for temporal and spatial property types is a behaviour changing feature, as the default type conversion is turned off in that case and dates aren't neither written nor read to and from strings.
```

Comment:
Double negative. Replace `aren't neither` with `are neither`.

Also replace `written nor read to and from` with `written to nor read from`.

Reply:
Wow. I'm ashamed I've written this 😆